cafe_information

```diff
@@ -0,0 +1,21 @@
+create table CafeInSanFrancisco (
```

Comment:
The table name must be written in plural form. In this case, I think `Cafes` will be nice. For `InSanFrancisco`, It is not useful because of your service design (actually, all cafes you provide to users are in San Francisco).

Reply:
Consider change `create table` to `CREATE TABLE`. Writing all SQL keywords by uppercase is suggested.

Reply:
I checked it. It would be better just using 'Cafes' as the table name. 
I think I have to change all keywords in uppercase. Thx